packages/common/mui/src/lib/faq/faq.tsx

```diff
@@ -33,6 +34,7 @@ export function Faq({
   componentAnswer,
   slug = '',
   sx,
+  onClick,
```

Comment:
Is it possible to make this more foolproof by importing the `trackEvent` function here and just pass in the two `type` and `title` props?